src/workspacer.Shared/Workspace/IWorkspaceManager.cs

```diff
@@ -8,10 +8,10 @@ namespace workspacer
 {
     public delegate void WorkspaceUpdatedDelegate();
     public delegate void FocusedMonitorUpdatedDelegate();
-    public delegate void WindowAddedDelegate(IWindow window, IWorkspace workspace);
-    public delegate void WindowUpdatedDelegate(IWindow window, IWorkspace workspace);
-    public delegate void WindowRemovedDelegate(IWindow window, IWorkspace workspace);
-    public delegate void WindowMovedDelegate(IWindow window, IWorkspace oldWorkspace, IWorkspace newWorkspace);
+    public delegate void WorkspaceWindowAddedDelegate(IWindow window, IWorkspace workspace);
+    public delegate void WorkspaceWindowUpdatedDelegate(IWindow window, IWorkspace workspace);
+    public delegate void WorkspaceWindowRemovedDelegate(IWindow window, IWorkspace workspace);
+    public delegate void WorkspaceWindowMovedDelegate(IWindow window, IWorkspace oldWorkspace, IWorkspace newWorkspace);
```

Comment:
I don't see the need to rename these delegates (or other types) and break existing configurations out there, unless there are valid reasons for doing so. Are changes required?